SpiritSpenderServer/SpiritSpenderServer/Controllers/ShotGlassPositionsController.cs

```diff
@@ -33,6 +33,16 @@ public async Task<ActionResult<long>> GetNumberOfPositions()
             return new ObjectResult(numberOfPositions);
         }
 
+        [HttpPut("clear")]
+        public async Task<ActionResult<IEnumerable<ShotGlassPositionSetting>>> GetClearPositions()
```

Comment:
Don't you also want to change the method name. No "get" needed

Reply:
Dont be so picky